pkg/kubefed2/unjoin.go

```diff
@@ -180,15 +190,18 @@ func UnjoinCluster(hostConfig, clusterConfig *rest.Config, federationNamespace,
 	}
 
 	if removeFromRegistry {
-		removeFromClusterRegistry(hostConfig, clusterNamespace, clusterConfig.Host, unjoiningClusterName, dryRun)
+		removeFromClusterRegistry(hostConfig, clusterNamespace, unjoiningClusterName, dryRun)
```

Comment:
(No action required) In future PRs, please ensure that unrelated non-functional changes are separated into their own commits to ease future maintenance.